app/src/layouts/organizationsControl/organizationsControl.scss

```
@@ -25,108 +25,171 @@
 
 .action {
   path {
-    fill: $COLOR--darkmode-topaz-text;
+    fill: var(--rp-ui-base-dark-topaz-text);
   }
 }
 
 .top-title {
-  @include truncate(200px);
+  @include truncate(100%);
+  min-width: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)